chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/UpdateMacraMeasuresJob.java

```diff
@@ -73,15 +92,27 @@ protected void doInTransactionWithoutResult(TransactionStatus status) {
     private static class UpdateMacraMeasuresDao extends BaseDAOImpl {
         private void updateSubstringInAllValues(final String oldSubstring, final String newSubstring) {
             getAllMeasuresWhereValueIsNotNull().stream()
-                .forEach(measure -> {
-                    measure.setValue(measure.getValue().replaceAll(oldSubstring, newSubstring));
-                    entityManager.merge(measure);
-                });
+                    .forEach(measure -> {
+                        measure.setValue(measure.getValue().replaceAll(oldSubstring, newSubstring));
+                        entityManager.merge(measure);
+                    });
+        }
+
+        private void updateStage2AsRemoved() {
+            getAllMeasuresWhereValueIsNotNull().stream()
+                    .filter(measure -> (measure.getCertificationCriterion().getNumber().equals(G_8)
+                            || measure.getCertificationCriterion().getNumber().equals(G_9))
```

Comment:
I know there is a Cures version of g9. I am not sure the order the jobs will be/need to be run but I just want to make sure that this code works as intended whether there is one g9 or two.